H3Lib/H3Lib.csproj

```diff
@@ -1,35 +1,36 @@
-<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I write this using JetBrains Rider, so I'll check to see how this works in that environment.